### PR TITLE
Docs - Promote

### PR DESCRIPTION
**Pull request by RakshithaK06**
Story details: https://app.clubhouse.io/kintohub/story/867

### CODE REVIEW

File: docs/anatomy/anatomy-releases.md [hunk 1]
```diff
@@ -67,6 +67,27 @@ Follow the below steps to tag your Release:
 
 ![Confirm Tag](/img/anatomy/confirm-tag.png)
 
+### Promote a Release
+
+KintoHub provides you an option to `Promote` your Releases across the different environments.
```
Comment (sumitkolhe) [inline docs/anatomy/anatomy-releases.md:72]: typo : remove "the". -> across different environments.

File: docs/anatomy/anatomy-releases.md [hunk 2]
```diff
@@ -67,6 +67,27 @@ Follow the below steps to tag your Release:
...
+
+You can do the following to **Promote** the release:
```
Comment (sumitkolhe) [inline docs/anatomy/anatomy-releases.md:74]: typo : promote a release

File: docs/anatomy/anatomy-releases.md [hunk 3]
```diff
@@ -67,6 +67,27 @@ Follow the below steps to tag your Release:
...
+
+1. Click on the `...` in the row of the `Tagged` release you wish to promote.
+2. Click on the **Promote Tag to Env** option
```
Comment (sumitkolhe) [inline docs/anatomy/anatomy-releases.md:77]: add full stop at the end

File: docs/anatomy/anatomy-releases.md [hunk 4]
```diff
@@ -67,6 +67,27 @@ Follow the below steps to tag your Release:
...
+
+![Promote Env](/img/anatomy/promote-env.png)
+
+3. Select the environment from the list you wish to promote the release in the prompt displayed.
```
Comment (sumitkolhe) [inline docs/anatomy/anatomy-releases.md:81]: Action : change sentence structure => In the displayed prompt, select the environment you wish to promote the release to.

File: docs/anatomy/anatomy-releases.md [hunk 5]
```diff
@@ -67,6 +67,27 @@ Follow the below steps to tag your Release:
...
+4. Click on the **Promote Release** button.
+
+![Confirm Promote](/img/anatomy/confirm-promote.png)
+
+:::note
+You cannot promote the release which is not tagged and the `Promote Tag to Env` option is disabled for the untagged releases.
```
Comment (sumitkolhe) [inline docs/anatomy/anatomy-releases.md:87]: typo : promote a release.
Actions : make the note in 2 points? 
1. You cannot promote a release which is not tagged.
2. the `Promote Tag to Env` option is disabled for untagged releases.

**Review by sumitkolhe — CHANGES_REQUESTED** [review]

File: docs/anatomy/anatomy-releases.md (hunk 2 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-releases.md:74]: Done

File: docs/anatomy/anatomy-releases.md (hunk 3 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-releases.md:77]: Done

File: docs/anatomy/anatomy-releases.md (hunk 4 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-releases.md:81]: Done

File: docs/anatomy/anatomy-releases.md (hunk 5 above)
Comment (RakshithaK06) [reply docs/anatomy/anatomy-releases.md:87]: Done

**Review by sumitkolhe — APPROVED** [review]

**Pull request merged**